frontend/src/components/EndpointOverviewCard/index.tsx

```diff
@@ -66,6 +121,36 @@ export default function EndpointOverviewCard(props: EndpointOverviewCardProps) {
           <span class={styles["overview__item-value"]}>{uptimeRate()}%</span>
         </div>
       </div>
+
+      <div class={styles.overview__content}>
+          <div class={styles["overview__content-item"]}>
+            <span class={styles["overview__item-label"]}>Message</span>
+            <span class={styles["overview__item-value"]}>{lastAdditionalMessage() ?? ""}</span>
+          </div>
+        {
+          lastHttpProtocol() != null &&
```

Comment:
i'm going to be nitpicky, please use Solid's control block because they look nicer

Reply:
what's solid control block